testing/kubelinter/lintcontext/parse_yaml.go

```diff
@@ -0,0 +1,243 @@
+package lintcontext
+
+import (
+	"bufio"
+	"bytes"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	y "github.com/ghodss/yaml"
+	"github.com/pkg/errors"
+	"github.com/rancher/charts/testing/kubelinter/k8sutil"
+	"github.com/rancher/charts/testing/scheme"
+	"helm.sh/helm/v3/pkg/chart"
+	"helm.sh/helm/v3/pkg/chart/loader"
+	"helm.sh/helm/v3/pkg/chartutil"
+	"helm.sh/helm/v3/pkg/cli/values"
+	"helm.sh/helm/v3/pkg/engine"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/apimachinery/pkg/util/yaml"
+)
+
+const (
+	// The max file size, in bytes, that we will load.
+	// TODO: make it configurable.
+	maxFileSizeBytes = 10 * 1024 * 1024
+)
+
+var (
+	clientSchema = scheme.Scheme()
```

Comment:
This was the only change that was required. Needed to replace `scheme.Scheme` (which is the default scheme without CRDs) with `scheme.Scheme()`, which points to a package that provides `scheme.Scheme` plus stuff.